lib/widget/interactive_page.dart

```diff
@@ -58,15 +59,38 @@ class _InteractivePageState extends State<InteractivePage>
       vsync: this,
       duration: const Duration(milliseconds: 200),
     );
+    _translateToCenterController = AnimationController(
+      vsync: this,
+      duration: const Duration(milliseconds: 200),
+    );
     super.initState();
   }
 
   @override
   void dispose() {
     transformationController.removeListener(transformListener);
+    _zoomAnimationController.dispose();
+    _zoomOutAnimationController.dispose();
+    _translateToCenterController.dispose();
     super.dispose();
   }
 
+  void animateDragPosition(double offsetY) {
+    final _offsetTween = Tween<double>(begin: offsetY, end: 0)
+        .animate(_translateToCenterController);
+    void animationListener() {
+      setState(() {
+        _dragPosition = Offset(0, _offsetTween.value);
+      });
+      if (_translateToCenterController.isCompleted) {
+        _translateToCenterController.removeListener(animationListener);
```

Comment:
I think the `_translateToCenterController` doesn't have a `animationListener` attached to it, right? Shouldn't it be `_offsetTween.removeListener(animationListener);`?

Reply:
Nice catch. Fixed it ✔️